pkg/client/results/reader.go

```diff
@@ -49,18 +49,22 @@ const (
 const (
 	// UnknownVersion lets the consumer know if this client can detect the archive version or not.
 	UnknownVersion = "v?.?"
-	versionEight   = "v0.8"
-	versionNine    = "v0.9"
-	versionTen     = "v0.10"
+	VersionEight   = "v0.8"
+	VersionNine    = "v0.9"
+	VersionTen     = "v0.10"
 )
 
 // Reader holds a reader and a version. It uses the version to know
-// where to find certain files within the archive stream.
+// where to find certain files within the archive.
 type Reader struct {
 	io.Reader
 	Version string
 }
 
+// NewReaderWithVersion will create a results.Reader that can interpret a
+// results archive of the version passed in.
+// This is useful when the reader can only be read once and the version of the
```

Comment:
Copyedit: "Useful if the reader can be read only once and if the version of the data to read is known."